MeetupPlatformApi/Migrations/20220401115515_MeetupPKFromIntToGuid.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace MeetupPlatformApi.Migrations
+{
+    public partial class MeetupPKFromIntToGuid : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropPrimaryKey(
+                name: "pk_meetups",
+                table: "meetups");
+
+            migrationBuilder.DropColumn(
+                name: "id",
+                table: "meetups");
+
+            migrationBuilder.AddColumn<Guid>(
```

Comment:
Will fail if we have at least 1 meetup in our database.

Reply:
Can 't add the data ? Or will the Up method terminate incorrectly ?